includes/class-sensei-course.php

```diff
@@ -205,18 +312,40 @@ public function add_custom_navigation() {
 			return;
 		}
 
+		$tabs       = $this->get_course_custom_navigation_tabs();
+		$screen_ids = wp_list_pluck( $tabs, 'screen_id' );
+
+		// Exclude admin pages (admin.php) because `in_admin_header` hook
+		// doesn't work properly for this case.
+		$screen_ids = array_filter( $screen_ids, [ $this, 'not_admin_page' ] );
```

Comment:
The `admin.php` has a different structure, so the default hook we're using works, but it causes many style issues, like making the WP Admin menu to be misaligned, for example.

And we also need a `wrap` div that doesn't exist in the `admin.php` pages, and we can introduce it through hooks in the correct places.